src/Contrl/Input.f90

```diff
@@ -386,7 +390,7 @@ subroutine in2_Input(onblem,oblength,obnseg,obmpstp,&
 
         end subroutine in2_Input
 
-        !> Input all parameters except beam line element parameters.
+        !> Input parameters for subsequent bunches.
```

Comment:
Not related to energy filtering, just clarifying the different subroutines in this module.